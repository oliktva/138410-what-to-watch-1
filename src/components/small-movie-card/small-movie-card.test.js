import React from 'react';
import renderer from 'react-test-renderer';

import SmallMovieCard from './small-movie-card';

const film = {
  id: 1,
  name: `Dardjeeling Limited`,
  img: `img/dardjeeling-limited.jpg`,
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  genre: `Adventure`
};

it(`renders correctly`, () => {
  const tree = renderer
    .create(
      <SmallMovieCard
        film={film}
        isPlaying={false}
      />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});
