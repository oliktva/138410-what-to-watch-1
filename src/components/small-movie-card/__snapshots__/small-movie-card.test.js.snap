// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
>
  <Link
    className="small-movie-card__link"
    to="/film/1"
  >
    <Videoplayer
      isPlaying={false}
      preview="https://es31-server.appspot.com/wtw/static/film/preview/pulp-fiction.jpg"
      src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
    />
    <h3
      className="small-movie-card__title"
    >
      Pulp Fiction
    </h3>
  </Link>
</article>
`;
